avocado/core/nrunner/__main__.py

```diff
@@ -0,0 +1,37 @@
+from avocado.core.nrunner.app import BaseRunnerApp
+from avocado.core.nrunner.registry import RUNNERS_REGISTRY_PYTHON_CLASS
+from avocado.core.runners.avocado_instrumented import \
+    AvocadoInstrumentedTestRunner
+from avocado.core.runners.dry_run import DryRunRunner
+from avocado.core.runners.exec_test import ExecTestRunner
+from avocado.core.runners.noop import NoOpRunner
+from avocado.core.runners.python_unittest import PythonUnittestRunner
+from avocado.core.runners.requirement_asset import RequirementAssetRunner
+from avocado.core.runners.requirement_package import RequirementPackageRunner
+from avocado.core.runners.sysinfo import SysinfoRunner
+from avocado.core.runners.tap import TAPRunner
+
+RUNNERS_REGISTRY_PYTHON_CLASS['dry-run'] = DryRunRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['noop'] = NoOpRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['exec-test'] = ExecTestRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['python-unittest'] = PythonUnittestRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['avocado-instrumented'] = AvocadoInstrumentedTestRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['requirement-asset'] = RequirementAssetRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['requirement-package'] = RequirementPackageRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['sysinfo'] = SysinfoRunner
+RUNNERS_REGISTRY_PYTHON_CLASS['tap'] = TAPRunner
```

Comment:
Hi @clebergnu is there any reason to fill the `RUNNERS_REGISTRY_PYTHON_CLASS` here instead of doing it inside each runner like [here](https://github.com/avocado-framework/avocado/pull/5191/files#diff-ba0d0822ad85c07d3ed2ba4798c3dd4d4fe969035c39771c82dab38326f903fcR163)? I think this will add a new step to runner's development process. Instead of just creating the runners based in the interface I, as a runner developer, will have to also do the update to the `avocado/core/nrunner/__main__.py`.

Reply:
+1 on that @richtja. If possible, would be nice to remove this from `__main__.py`.